src/components/item/settings/ItemLoginSetting.js

```diff
@@ -53,15 +53,17 @@ const ItemLoginSwitch = ({ item }) => {
 
   // update state variables depending on fetch values
   useEffect(() => {
-    const tagValue = getItemLoginTagFromItem({ tags, itemTags });
-    setItemLoginTagValueForItem(tagValue);
-    setIsItemLoginEnabled(Boolean(tagValue));
-    // disable setting if it is set on a parent
-    setIsSwitchDisabled(tagValue && tagValue?.itemPath !== item?.get('path'));
-    setSchema(
-      getItemLoginSchema(item?.get('extra')) ||
-        SETTINGS.ITEM_LOGIN.OPTIONS.USERNAME,
-    );
+    if (tags && itemTags) {
+      const tagValue = getItemLoginTagFromItem({ tags, itemTags });
+      setItemLoginTagValueForItem(tagValue);
+      setIsItemLoginEnabled(Boolean(tagValue));
+      // disable setting if it is set on a parent
```

Comment:
Not sure what this means. Does this apply to all ancestors or just the immediate parent?

Reply:
Yes, if the tag exists it is either because the current object or one of its parents has the tag. The check on the path is valid as long as the path is not the current item's.